hemres/siterelated.py

```diff
@@ -0,0 +1,98 @@
+""" Copyright Mezzanine """
```

Comment:
Maybe add a comment why this is here? I think it should be clear what is production code and what is not. Unless this is also meant to be used in production.